### ISSUE TITLE
model in n3-checkbox-group and n3-radio-group

### ISSUE THREAD

**Issue by Zutatensuppe** [N3-components/N3-components#65]
hi,

i noticed that in `n3-checkbox-group` and `n3-radio-group` the model is not used when initializing the component. i have to set `selected` attribute manually. 
is this desired or can it be fixed?

```
<template>
<n3-checkbox-group v-model="checklist">
  <n3-checkbox label="one">one</n3-checkbox>
  <n3-checkbox label="two">two</n3-checkbox>
</n3-checkbox-group>
{{checklist}} -- corresponds to the checked checkboxes
</template>
<script>
export default {
  data() {
    checklist: ['one'], // this will not set first checkbox to checked. :(
  }
}
</script>
```

**Comment by ada012** [comment]
@Zutatensuppe Ok, let's see.

**Comment by x-cold** [comment]
This bug has been fixed in a4dc4512377a9b5c7a4bcf47167ab52cd34722a5.
